### DISCUSSION TITLE
MacOS Quartz graphics difference

### DISCUSSION THREAD

**Discussion by rageworx** [fire-eggs/YAIV#28] [General]
Actually it almost same on Linux, there's some default graphics are different with Windows GDI.
See this unittest, line drawing.

<img width="812" alt="image" src="https://user-images.githubusercontent.com/6948225/119623066-c5c3e200-be42-11eb-9f6b-c999af8aa940.png">

There's many different graphics quality, font hinting, antialiased clean lines ... perfect high DPI.

**Comment by rageworx** [comment]
line drawing case,

![image](https://user-images.githubusercontent.com/6948225/119623907-9792d200-be43-11eb-8003-84aab4679eed.png)

**Reply by rageworx** [reply]
Windows GDI.

![image](https://user-images.githubusercontent.com/6948225/119624637-4f27e400-be44-11eb-9f97-ddaf77d7f35b.png)

**Reply by rageworx** [reply]
Linux x86.64 ( Mint Linux 18.3 silvia ) , cinamon

![image](https://user-images.githubusercontent.com/6948225/119627096-d5ddc080-be46-11eb-8a19-383739f92e60.png)

**Reply by rageworx** [reply]
Linux aarch64 ( debian, LXDE ) , openbox

![image](https://user-images.githubusercontent.com/6948225/119627764-7633e500-be47-11eb-8406-6f7e216864e3.png)

**Reply by fire-eggs** [reply]
Amazing!
